### ISSUE TITLE
java.util.MissingResourceException: Can't find bundle for base name messages.PoetryPsiBundle

### ISSUE THREAD

**Issue by vlasovskikh** [koxudaxi/poetry-pycharm-plugin#115]
**To Reproduce**
1. Start PyCharm with a project opened
2. Wait for the IDE to initialise
2. Observe the red exclamation mark error in the right corner of the status bar, the exception is:

```
java.util.MissingResourceException: Can't find bundle for base name messages.PoetryPsiBundle, locale ru_RU
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2055)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1689)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1593)
	at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:1284)
	at com.intellij.AbstractBundle.getResourceBundle(AbstractBundle.java:156)
	at com.intellij.AbstractBundle.getResourceBundle(AbstractBundle.java:143)
	at com.intellij.codeInspection.InspectionEP.getLocalizedString(InspectionEP.java:200)
	at com.intellij.codeInspection.InspectionEP.getGroupDisplayName(InspectionEP.java:61)
	at com.intellij.codeInspection.InspectionEP.getGroupPath(InspectionEP.java:132)
	at com.intellij.codeInspection.ex.InspectionToolWrapper.getGroupPath(InspectionToolWrapper.java:156)
	at com.intellij.codeInspection.ex.InspectionSearchableOptionContributor.processOptions(InspectionSearchableOptionContributor.java:40)
	at com.intellij.ide.ui.search.SearchableOptionsRegistrarImpl.lambda$initialize$1(SearchableOptionsRegistrarImpl.java:131)
	at com.intellij.openapi.extensions.impl.ExtensionProcessingHelper.forEachExtensionSafe(ExtensionProcessingHelper.java:33)
	at com.intellij.openapi.extensions.ExtensionPointName.forEachExtensionSafe(ExtensionPointName.java:49)
	at com.intellij.ide.ui.search.SearchableOptionsRegistrarImpl.initialize(SearchableOptionsRegistrarImpl.java:131)
	at com.intellij.openapi.options.newEditor.SettingsFilter.lambda$new$1(SettingsFilter.java:53)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Environments (please complete the following information):**
 - IDE: PyCharm Pro 2020.3 EAP 1
 - Plugin version: 0.5.1

**Additional context**

It seems there is an unresolved reference to this bundle in:

https://github.com/koxudaxi/poetry-pycharm-plugin/blob/21fffd85adf54b813149818a84d037c3d01fe04e/resources/META-INF/only-toml.xml#L7

**Comment by koxudaxi** [comment]
@vlasovskikh 
I have fixed and pushed it to JetBrains Market. :rocket: 

Thank you very much.

**Comment by koxudaxi** [comment]
@vlasovskikh 
It's published as `0.5.2`

**Comment by vlasovskikh** [comment]
@koxudaxi Thanks! I've updated to 0.5.2 and I no longer see the exception.
